### ISSUE TITLE
attributes: How to access `[[deprecated(str)]]`?

### ISSUE THREAD

**Issue by EricCousineau-TRI** [CastXML/pygccxml#122]
Is it possible to access `[[deprecated(str)]]`, without using `__attribute__((gccxml(”...”)))`?

From the current docs, it looks like it only exposes the custom attributes?
https://pygccxml.readthedocs.io/en/master/apidocs/pygccxml.declarations.declaration.html#pygccxml.declarations.declaration.declaration_t.attributes
> ![image](https://user-images.githubusercontent.com/26719449/87235060-d17ef500-c3a5-11ea-820c-f768101b7b64.png)

---

Part of https://github.com/RobotLocomotion/drake/issues/13670

Example:
https://github.com/RobotLocomotion/drake/blob/0d219dbc2642ce6f9c8120cb5839ac87a5bc637d/tools/workspace/pybind11/test/sample_header.h#L260
https://github.com/RobotLocomotion/drake/blob/0d219dbc2642ce6f9c8120cb5839ac87a5bc637d/common/drake_deprecated.h#L59-L63

\cc @jamiesnape @josephsnyder

**Comment by EricCousineau-TRI** [comment]
Upon inspection, at least some of the info is exposed:
https://gist.github.com/EricCousineau-TRI/8da1ad7963fa58315db5852488e18922#file-drake_issue13670_pygccxml_mkdoc_output-txt-L591-L599
```
class_t: 'DrakeDeprecatedClass'

    location: [bazel-out/k8-opt/bin/tools/workspace/pybind11/_virtual_includes/sample_header/drake/tools/workspace/pybind11/test/sample_header.h]:261
    artificial: 'False' 
    attributes: deprecated
    class type: 'class' 
    size: 1
    align: 1
    public:
```

Will see if it includes the information contained within the deprecation too.

That being said, seems like the docs are stale?

**Comment by EricCousineau-TRI** [comment]
I see that `attributes` are tested here:
https://github.com/CastXML/CastXML/blob/8625aed52c8e2e0b94cfd65767a7564a36ce4c96/test/input/Class-deprecated.cxx
https://github.com/CastXML/CastXML/blob/8625aed52c8e2e0b94cfd65767a7564a36ce4c96/test/expect/castxml1.any.Class-deprecated.xml.txt#L3

However, it doesn't test for text (but the test doesn't even show it, so that's understandable)...

**Comment by EricCousineau-TRI** [comment]
Will upstream this issue to CastXML when it makes sense.

**Comment by EricCousineau-TRI** [comment]
Resolved by #138
